packages/react/test/specs/lib/mergeThemes/mergeThemeVariables-test.ts

```diff
@@ -0,0 +1,48 @@
+import { mergeThemeVariables } from 'src/lib/mergeThemes'
+import * as _ from 'lodash'
+import { withDebugId } from 'src/lib'
+
+describe('mergeThemeVariables', () => {
+  test('component variables are merged', () => {
+    const target = { Button: {} }
+    const source = { Icon: {} }
+
+    const merged = mergeThemeVariables(target, source)
+
+    expect(merged).toHaveProperty('Button')
+    expect(merged).toHaveProperty('Icon')
+  })
+
+  test('component variable objects are converted to functions', () => {
+    const target = { Button: {} }
+    const source = { Button: {} }
+
+    const merged = mergeThemeVariables(target, source)
+
+    expect(merged.Button).toBeInstanceOf(Function)
+    expect(merged.Button).toBeInstanceOf(Function)
+  })
+
+  xtest('component variable objects are deeply merged', () => {
```

Comment:
```suggestion
  test('component variable objects are deeply merged', () => {
```

Should be reenabled?

Reply:
reenabled